### PR TITLE
Namespace Selector in MlFlow UI

### PR DESCRIPTION
**Pull request by fabiovincenzi**
Fixes https://github.com/G-Research/fasttrackml/issues/463.
Introduces namespace selector in the mlflow UI.
Screenshot 2024-01-20 at 20 16 54
Screenshot 2024-01-20 at 20 17 01

### CODE REVIEW

File: src/src/experiment-tracking/components/App.tsx [hunk 1]
```diff
@@ -56,9 +57,28 @@ const classNames = {
...
-  state = {
-    version: 'unknown',
+  state: {
+    selectedNamespace: string; namespaces: string[];
+    version: string;
   };
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      selectedNamespace: 'ns1',
```
Comment (dsuhinin) [inline src/src/experiment-tracking/components/App.tsx]: why `selectedNamesapce` in state initially -> `ns1`? debug?

File: src/src/experiment-tracking/components/App.tsx [hunk 2]
```diff
@@ -56,9 +57,28 @@ const classNames = {
...
+      namespaces: ['cccccc787493', 'ddd', 'ns1', 'ns2'],
```
Comment (dsuhinin) [inline src/src/experiment-tracking/components/App.tsx]: the same question here. probably should be an empty array?

**Review by dsuhinin — CHANGES_REQUESTED** [review]

File: src/src/experiment-tracking/components/App.tsx (hunk 1 above)
Comment (fabiovincenzi) [reply src/src/experiment-tracking/components/App.tsx]: Ops, sorry I'll fix it

File: src/src/experiment-tracking/components/App.tsx (hunk 2 above)
Comment (fabiovincenzi) [reply src/src/experiment-tracking/components/App.tsx]: Yep, same

**Review by dsuhinin — APPROVED** [review]
Looks good!

**Comment by fabiovincenzi** [comment]
Introduces namespace selector in the mlflow UI.

**Pull request merged**